packages/edit-site/src/store/actions.js

```diff
@@ -298,3 +300,140 @@ export function setIsListViewOpened( isOpen ) {
 		isOpen,
 	};
 }
+
+/**
+ * Reverts a template to its original theme-provided file.
+ *
+ * @param {Object} template The template to revert.
+ */
+export function* revertTemplate( template ) {
```

Comment:
Note that all the dispatches here fail the `@wordpress/data-no-store-string-literals` lint rule.